packages/crayons-core/src/index.html

```diff
@@ -13,6 +13,25 @@
   </head>
   <body>
     <fw-button expand color="primary">Primary Button</fw-button>
+    <!-- <fw-popover trigger="hover" hide-after="2000" show-after="2000">
```

Comment:
Can you please revert the changes in index.html ? We are good to go otherwise!

Reply:
Sure, will do